src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -423,7 +423,9 @@
             validateWorkflow() {
                 const formData = new FormData()
                 formData.append("file", this.importedFile)
-                const params = `?branch=${this.workflowMetadata.git_branch}&commit=${this.workflowMetadata.git_commit}`
+                const branch = this.workflowMetadata.git_branch ? `${this.workflowMetadata.git_branch}` : '';
+                const commit = this.workflowMetadata.git_commit ? `${this.workflowMetadata.git_commit}` : '';
+                const params = `?branch=${branch}&commit=${commit}`
```

Comment:
I think you should be able to use this slightly terser pattern:

```suggestion
                const branch = this.workflowMetadata.git_branch || '';
                const commit = this.workflowMetadata.git_commit || '';
                const params = `?branch=${branch}&commit=${commit}`
```